plugins/scribe-systemd/src/Cardano/BM/Scribe/Systemd.lhs

```diff
@@ -58,7 +58,7 @@ plugin :: (IsEffectuator s a, ToJSON a, FromJSON a)
        => Configuration -> Trace IO a -> s a -> IO (Plugin a)
 plugin _ _ _ =
     ScribePlugin
-               <$> mkJournalScribe
+               <$> mkJournalScribe "cardano"
```

Comment:
what if the identifier can be passed in as an argument?

Reply:
@CodiePP, done.